recipes/feature_merge/meta.yaml

```diff
@@ -21,6 +21,7 @@ requirements:
     - pbr
```

Comment:
pbr can be removed here I guess

Reply:
Looks like it is required for the build step.